engine/engine.go

```diff
@@ -135,13 +145,38 @@ func cmd() *cobra.Command {
 			cmd.Println("")
 
 			// printout in PEM
-			inPem, err := cc.GetPublicKeyAsPEM(types.KeyForEntity(le))
-			if err != nil {
+			if publicKey, err := jwk.Materialize(); err != nil {
```

Comment:
these extra changes make this PR longer than needed, better to do them apart. This PR is already 2k+ lines long

Reply:
I changed this to avoid having to implement GetPublicKeyAsPEM in the API client as well, so it's about choosing to optimize this (the extra roundtrip to the server is unnecessary) v.s. implementing GetPublicKeyAsPEM in the API client (since I only implemented stuff there that's actually being used).